examples/xgboost/custom/xgboost_model_persistor.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_constant import FLContextKey
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.abstract.model import ModelLearnable, make_model_learnable
+from nvflare.app_common.abstract.model_persistor import ModelPersistor
+
+
+class XGBoostModelPersistor(ModelPersistor):
+    def __init__(self, save_name="xgboost_model.json"):
```

Comment:
This file can be called "JSONModelPersistor", as it only uses JSON to save/load data.

Reply:
sounds good!